webpack/JobWizard/autofill.js

```diff
@@ -76,7 +76,7 @@ export const useAutoFill = ({
             if (typeof rest[key] === 'string') {
               setTemplateValues(prev => ({ ...prev, [input]: rest[key] }));
               setAdvancedValues(prev => ({ ...prev, [input]: rest[key] }));
-            } else {
+            } else if (rest[key].value !== null) {
```

Comment:
is only `null` values are invalid? or should it be `else if (rest[key].value)` ?

Reply:
I wanted to be as specific as possible, without catching more general falsey values. Not sure if we can get any apart from `false`